src/sagas/wallet.js

```diff
@@ -250,6 +252,13 @@ export function* startWallet(action) {
     });
     console.log('[+] Start wallet.', serverInfo);
 
+    // Use HTR as default
+    let nativeToken = hathorLibConstants.DEFAULT_TOKEN_CONFIG;
+    if (info.native_token) {
+      nativeToken = info.native_token;
+    }
```

Comment:
Can we use `wallet.storage.getNativeTokenData()`?

Reply:
Done